docs/keyConcepts.md

```diff
@@ -61,6 +61,12 @@ or [Prometheus](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.h
 VictoriaMetrics supports enforcing of label filters for [query API](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#prometheus-querying-api-enhancements)
 to emulate data isolation. However, the real data isolation can be achieved via [multi-tenancy](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#multitenancy).
 
+VictoriaMetrics stores all the values as `float64`, 
+so these values can lose precision when the value contains more than `12-15` decimal digits.
+Initially, it called `strconv.ParseFloat(str, 64)`. This loses precision in lower 3 decimal digits.
+Input integer values in the range `[-2^54 ... 2^54-1]` shouldn't lose precision, while all the integer values outside this 
+range may lose precision due to conversions explained above.
```

Comment:
```suggestion
VictoriaMetrics stores all the values as [float64](https://en.wikipedia.org/wiki/Double-precision_floating-point_format) values with [extra compression](https://faun.pub/victoriametrics-achieving-better-compression-for-time-series-data-than-gorilla-317bc1f95932) applied.
This guarantees precision correctness for values with up to 12 significant decimal digits ([-2^54 ... 2^54-1]). 
```

I believe this comment should belong to https://docs.victoriametrics.com/keyConcepts.html#raw-samples

Reply:
updated. moved the description to the raw-samples